portfolio/src/main/java/com/google/sps/servlets/AuthenticationURLServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+/* Servlet that adds a comment to the dataserve. */
```

Comment:
This seems to describe a different Servlet.

Reply:
Whoops, thanks.